trust/meson.build

```diff
@@ -105,6 +105,11 @@ executable('trust',
                           libp11_tool_dep] + libffi_deps + dlopen_deps + libtasn1_deps,
            install: true)
 
+configure_file(input: 'trust-extract-compat.in',
+               output: 'trust-extract-compat',
+               configuration: configuration_data(),
```

Comment:
It's not a problem in your patch, but it looks like `trust-extract-compat.in` doesn't contain any substitution keywords (i.e., @...@). Maybe we can just rename `trust-extract-compat.in` to `trust-extract-compat` in the repo?

Reply:
I agree.  New version force-pushed.